scripts/prepdocs.py

```diff
@@ -70,7 +71,7 @@ async def setup_file_strategy(credential: AsyncTokenCredential, args: Any) -> St
     sentence_text_splitter = SentenceTextSplitter(has_image_embeddings=args.searchimages)
     file_processors = {
         ".pdf": FileProcessor(pdf_parser, sentence_text_splitter),
-        ".html": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".html": FileProcessor(LocalHTMLParser(), sentence_text_splitter),
```

Comment:
I think we need to do this similar to how we do the pdf parser - add an option for using the local parser - with the default of using Document Intelligence. It's a pain adding new options since you have to modify the sh/ps1 scripts, but it's important to make it an option. If you have other ideas for switching between the two, let me know.

Reply:
will review and see if I don't find anything better will implement it